examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/scenario/ExampleScenarioFactory.java

```diff
@@ -86,10 +90,39 @@ private FrameworkExampleScenario getFrameworkScenario(final String framework) {
     public Map<String, String> getResourceTemplateMap() {
         Map<String, String> result = new HashMap<>();
         for (FeatureExampleScenario each : featureScenarios) {
-            result.putAll(each.getJavaClassTemplateMap());
+            result.putAll(each.getResourceTemplateMap());
         }
         result.putAll(frameworkScenario.getResourceTemplateMap());
         result.put("resources/logback.ftl", "logback.xml");
         return result;
     }
+    
+    /**
+     * Get java class path collection.
```

Comment:
Please change java from `path collection` to `paths`

Reply:
done.